connector_sage/migrations/14.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2022 ForgeFlow S.L.  <https://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# pylint: disable=C7902
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(env.cr, "connector_sage", "14.0.1.0.0/noupdate_changes.xml")
```

Comment:
```suggestion
    openupgrade.load_data(
        env.cr, "connector_sage", "migrations/14.0.1.0.0/noupdate_changes.xml"
    )
```
sorry, an oversight. Fixed in https://github.com/nuobit/odoo-addons/pull/138.